packages/core/src/server.ts

```diff
@@ -9,13 +9,11 @@ import {Context, Binding, Constructor} from '@loopback/context';
  * Defines the requirements to implement a Server for LoopBack applications:
  * start() : Promise<void>
  * stop() : Promise<void>
- * Additionally, an implementation must create a child context instance from
- * the application
- * ```ts
- * constructor(app: Application, options?: {}, ...args: any[]) {
- *   app.createChildContext();
- * }
- * ```
+ * It is recommended that each Server implementation creates its own child
+ * Context, which inherits from the parent Application context. This way,
+ * any Server-specific bindings will remain local to the Server instance,
+ * and will avoid polluting global module scope.
```

Comment:
I find "module scope" misleading - the parent scope is at application-level. A module is allowed to have more than one application, there is no global module-level scope.